TypeRaceAir/race.py

```diff
@@ -0,0 +1,63 @@
+import typing
+
+
+class Race():
+
+    def __init__(self, sentece: str, players: typing.List[Player]):
+        self.sentece = sentece
+        self.players = players
+
+
+class Player():
+    pass
+
+
+class PlayerRaceResult():
+    def __init__(self, race: Race, player: Player, submit_timestamp: float, submit_sentence: str):
+        self.race = race
+        self.submit_sentence = submit_sentence
+        self.player = player
+        self.submit_timestamp = submit_timestamp
+
+    @property
+    def words_mistake(self,) -> int:
+        '''The number of words that the user typed incorrectly. '''
+        return len(self.race.sentece.split()) - self.words_correct
+
+    @property
+    def words_correct(self,) -> int:
+        '''The number of words that the user typed correctly. '''
+        submitted_words = self.submit_sentence.split()
+        target_words = self.race.sentece.split()
+        return sum(
+            1
+            for cur_submitted, cur_target in zip(submitted_words, target_words)
+            if cur_submitted == cur_target
+        )
+
+    @ property
+    def accuracy(self,) -> float:
+        '''The percentage of words that the user typed correctly, out of the total words in the sentence. '''
+        total_words = len(self.race.sentece.split())
+        return self.words_correct/total_words
+
+    @ property
+    def time(self,) -> float:
+        '''The time that it took for the user to submit his answer, in seconds. '''
+        return self.submit_timestamp - self.race.start_timestamp
+
+    @ property
+    def wpm(self,) -> float:
+        '''The WPM (Words Per Minute) of the current race for the current user. '''
+        total_words = len(self.race.sentece.split())
+        return (60/self.time)*total_words
+
+    @ property
+    def cwpm(self,) -> float:
+        '''The CWPM (Correct Words Per Minute) of the current race for the current user. '''
+        return (60/self.time)*self.words_correct
+
+    @ property
+    def score(self,) -> int:
+        '''The total calculated score of the current player in the current race. '''
+        return
```

Comment:
**???**
If not implemented yet, use the `pass` keyword (or even better - don't check-in)